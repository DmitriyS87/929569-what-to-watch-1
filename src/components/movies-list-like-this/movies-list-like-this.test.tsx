import * as React from 'react';
import * as renderer from 'react-test-renderer';
import MoviesListLikeThis from './movies-list-like-this';

it(`MoviesListLikeThis correctly renders after relunch`, () => {
  const tree = renderer.create(
    <MoviesListLikeThis
      children={<div>Mock Children</div>}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

