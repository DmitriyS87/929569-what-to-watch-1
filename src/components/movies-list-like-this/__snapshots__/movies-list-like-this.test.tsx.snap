// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesListLikeThis correctly renders after relunch 1`] = `
<section
  className="catalog catalog--like-this"
>
  <h2
    className="catalog__title"
  >
    More like this
  </h2>
  <div>
    Mock Children
  </div>
</section>
`;
